### PR TITLE
chore(docs): match doc page titles with their respective titles in toc

### PR DESCRIPTION
**Pull request by linostar**
- [X] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

I changed the doc page titles, in tutorials and how-to guides, in order to match them with their respective titles in table of contents.

### CODE REVIEW

**Comment by linostar** [comment]
@cjdcordeiro Ready for review.

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
Thanks for this.

It's a shame that some of these names are quite long and thus force the navigation links to break into 2 lines. could you try "crafting" some titles in the hope we can keep them under a single line? 

I know it won't work for all of them (like 4. and 5.), but for ex., I think we can keep the `Hello World` as `Hello World`, and maybe there's some better&shorter wording we can use for others? 

It's a nitpick, and if you can't find shorter titles, so be it :)

**Comment by linostar** [comment]
@cjdcordeiro I shortened the ones I changed, and some of the unchanged ones too. Let me know what you think.

File: docs/how-to/index.rst [hunk 1]
```diff
@@ -14,12 +14,12 @@ adapt the steps to fit your specific requirements.
    :maxdepth: 1
 
    Get started - quick guide <get-started>
-   Build a ROCK with GitHub Action <rockcraft-pack-action>
+   Use GitHub Action to ROCK <rockcraft-pack-action>
```
Comment (cjdcordeiro) [inline docs/how-to/index.rst:17]:
```suggestion
   Use Rockcraft's GitHub Action <rockcraft-pack-action>
```

File: docs/how-to/rockcraft-pack-action.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
-How to build a ROCK with Rockcraft's GitHub Action
-**************************************************
+How to Use GitHub Action to ROCK
```
Comment (cjdcordeiro) [inline docs/how-to/rockcraft-pack-action.rst:1]:
```suggestion
How to use Rockcraft's GitHub Action
```

File: docs/tutorials/chisel.rst [hunk 3]
```diff
@@ -1,5 +1,5 @@
-Install chiselled slices into a ROCK
-====================================
+Put Chisel slices in a ROCK
```
Comment (cjdcordeiro) [inline docs/tutorials/chisel.rst:1]:
```suggestion
Install package slices in a ROCK
```

File: docs/tutorials/index.rst [hunk 4]
```diff
@@ -10,8 +10,8 @@ reproducible steps.
 .. toctree::
    :maxdepth: 1
 
-   1. Create a "Hello World" ROCK <hello-world>
-   2. Create a ROCK for a PyPI package <pypi-package.rst>
-   3. Install chiselled packages into a ROCK <chisel.rst>
+   1. Create a Hello World ROCK <hello-world>
+   2. Containerise a PyPI package <pypi-package.rst>
+   3. Put Chisel slices in a ROCK <chisel.rst>
```
Comment (cjdcordeiro) [inline docs/tutorials/index.rst:15]:
```suggestion
   3. Install package slices in a ROCK <chisel.rst>
```

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: docs/how-to/index.rst (hunk 1 above)
Comment (linostar) [reply docs/how-to/index.rst:17]: OK

File: docs/how-to/rockcraft-pack-action.rst (hunk 2 above)
Comment (linostar) [reply docs/how-to/rockcraft-pack-action.rst:1]: Ok

File: docs/tutorials/chisel.rst (hunk 3 above)
Comment (linostar) [reply docs/tutorials/chisel.rst:1]: How about `Install Chisel slices in a ROCK` then? it would fit in one line.
